models/marts/tests/intermediate/int_model_test_summary.sql

```diff
@@ -59,7 +95,8 @@ final as (
         all_graph_resources.resource_type,
         all_graph_resources.model_type,
         coalesce(agg_test_relationships.is_primary_key_tested, FALSE) as is_primary_key_tested,
-        coalesce(agg_test_relationships.number_of_tests_on_model, 0) as number_of_tests_on_model
+        coalesce(agg_test_relationships.number_of_tests_on_model, 0) as number_of_tests_on_model,
+        coalesce(agg_test_relationships.number_of_constraints_on_model, 0) as number_of_constraints_on_model
```

Comment:
for consistency's sake, we may want to full this through to the `fct_` model as well! not blocking

Reply:
Hmm, which `fct_` model are you thinking of? I believe the column already makes its way into `fct_missing_primary_key_tests`, courtesy of a few `select *` statements.